misc/create_burnman_readable_perplex_table.py

```diff
@@ -0,0 +1,86 @@
+from subprocess import Popen, PIPE, STDOUT
+import argparse
+
+'''  
+This python file uses PerpleX's werami software 
```

Comment:
werami is quite slow, so this is certainly not something I would use frequently, but you're right. 
I'll add a function to perplex.py.